openassessment/templates/openassessmentblock/staff_area/staff_area.html

```diff
@@ -103,7 +103,7 @@ <h2 class="staff-info__title ui-staff__title">
                 </div>
 
                 <div class="staff-info__status ui-staff__content__section">
-                    <table class="staff-info__status__table" summary="{% trans "Where are your learners currently in this problem" %}">
```

Comment:
Reviewers: @catong thought this summary was confusing, and in our investigations it appears that the attribute has been deprecated anyway:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table